Terraform/Home_Task_2/modules/alb/main.tf

```diff
@@ -49,13 +39,36 @@ resource "aws_alb_listener" "listener_http" {
     target_group_arn = aws_alb_target_group.this_tg.arn

```

Comment:
port and protocol parameters (resource "aws_alb_listener") - define values of these parameters in modules/alb/variables.tf file